docs/web-chat/api.mdx

```diff
@@ -137,6 +137,151 @@ You should call this API before a conversation starts. In other words, it works
 
 </Admonition>
 
+## Login {#login}
```

Comment:
Can we add all the new identity system related pieces in one single place for easy reference similar to SDK dev docs. Customers might find it very difficult like this.?